test/kwtool/accounts_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule Kwtool.AccountsTest do
+  use Kwtool.DataCase
+
+  alias Kwtool.Accounts
+  alias Kwtool.Accounts.User
+
+  describe "changeset/2" do
+    test "list_users/0 returns all users" do
+      user = insert(:user)
+      assert Accounts.list_users() == [user]
+    end
+
+    test "get_user!/1 returns the user with given id" do
+      user = insert(:user)
+      assert Accounts.get_user!(user.id) == user
+    end
+
+    test "create_user/1 with valid data creates a user" do
+      valid_users = params_for(:user, %{password: "123456", password_confirmation: "123456"})
+      assert {:ok, %User{} = user} = Accounts.create_user(valid_users)
```

Comment:
To make the test easier to read, how about rename `user` variable to `user_in_db` or `created_user`?

Reply:
Updated